iphone/Maps/Core/Theme/Core/ThemeManager.swift

```diff
@@ -65,6 +69,22 @@ final class ThemeManager: NSObject {
     instance.update(theme: Settings.theme())
   }
 
+  @available(iOS 13.0, *)
+  private func updateSystemUserInterfaceStyle(_ theme: MWMTheme) {
+    let userInterfaceStyle: UIUserInterfaceStyle = { theme in
+      switch theme {
+      case .day: fallthrough
+      case .vehicleDay: return .light
```

Comment:
Outdoors style is handled differently, right?

Reply:
Yes!It is not part of themes enum. And not affected by this change.